services/spar/src/Spar/Scim/Auth.hs

```diff
@@ -41,14 +41,17 @@ import Data.String.Conversions (cs)
 import Data.Time (getCurrentTime)
 import Imports
 import OpenSSL.Random (randBytes)
+-- FUTUREWORK: these imports are not very handy.  split up Spar.Scim into
+-- Spar.Scim.{Core,User,Group} to avoid at least some of the hscim name clashes?
```

Comment:
ormolu is not very good at leaving comments in peace, especially in the import section.  but this will do, it's clear what it refers to.